ui/tests/pages/components/facet.js

```diff
@@ -44,3 +44,16 @@ export const singleFacet = (scope) => ({
     },
   }),
 });
+
+export const hdsFacet = (scope) => ({
+  scope,
+
+  toggle: clickable('.hds-dropdown-toggle-button'),
+
+  options: collection('.hds-dropdown-list-item', {
+    resetScope: true,
+    label: text(),
+    key: attribute('data-test-hds-facet-option'),
+    toggle: clickable('.hds-dropdown-list-item__label'),
+  }),
+});
```

Comment:
A good chunk of work here is figuring out the nuances between the PowerSelect components we were using below for dropdowns, and the new Helios ones. You'll see `key` change to `label` throughout the test files generally, among other things.